### ISSUE TITLE
Inserter blocks focus style

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#1355]
At some point, the blocks in the inserter had an accessible focus style (a subtle border), see screenshot on the left. That was discussed at length and agreed to be a good balance between design and accessibility.

I see now on master the focus style has changed (see screenshot on the right) and it's back to just a color change:

<img width="738" alt="screen shot 2017-06-22 at 12 01 30" src="https://user-images.githubusercontent.com/1682452/27429583-d662d0ca-5745-11e7-8f59-1490c195baf4.png">

Whatever the final decision will be, I'd like to remind color alone is not sufficient.

**Comment by jasmussen** [comment]
I'm probably to blame for the regression, as the mockups were insufficient here. Instead of the glowing blue outline, let's go back to the previous style, which is similar to that of buttons in the quick toolbar.

**Comment by jasmussen** [comment]
How would you feel about this focus style for the blocks?
 https://github.com/WordPress/gutenberg/issues/1325#issuecomment-311594148

**Comment by afercia** [comment]
Sorry for late reply.  https://github.com/WordPress/gutenberg/issues/1325#issuecomment-311594148 looks good to me, assuming the blue has a sufficient contrast ratio with the background.

Adding to the list: the "Continue Writing" items would need a better focus style too:

<img width="362" alt="screen shot 2017-07-22 at 11 42 52" src="https://user-images.githubusercontent.com/1682452/28490182-b6af9ca4-6ed3-11e7-98f0-5d98b5cbcb27.png">

**Comment by afercia** [comment]
As a reference, the focus style was incidentally removed [here](https://github.com/WordPress/gutenberg/commit/2ba4040a13f7c8f704065dd3092ed25674507cc7) see #835

**Comment by afercia** [comment]
Marking with the high priority label to indicate it's an a11y priority.
